CHANGELOG.md

```diff
@@ -70,7 +198,6 @@ Big thanks to the 19 contributors who made this release possible. Here are some
 - <!-- 15 --> [test] Add current behavior for a11y name vs visible name for PickersDay (#27661) @eps1lon
 - <!-- 17 --> [test] Dodge double logging in dev mode (#27653) @oliviertassinari
 - <!-- 14 --> [test] Enable skipped test fixed by upstream React fix (#27615) @eps1lon
-- <!-- 19 --> [theme] Add missed variants in Components interface (#27453) @nikitabobers
```

Comment:
What's the reasoning here? We tagged this change for `beta.4`: https://github.com/mui-org/material-ui/commit/fc33c8170b0df49c68307d2c1c6cecfd5f878fe5. Was it not actually part of the beta.4 release?

Reply:
My bad, I must have moved the incorrect line. Thanks for pointing out will revert.